README.md

```diff
@@ -79,14 +79,14 @@ mosquitto:
        path: /mosquitto/tls/server.key
        dest: /etc/mosquitto/certs/server.key
        mode: "0600"
-  # All key/value pairs under mosquitto.settings will be added to 
+  # All key/value pairs under sansible_mosquitto_settings will be added to 
   # `/etc/mosquitto/conf.d/mosquitto.local.conf`
   settings:
     allow_anonymous: "true"
     persistence: "true"
   user_list:
     # Format is username: password, passwords must be generated using mosquitto_passwd
-    # mosquitto.settings.password_file must be defined if you want to enable users
+    # sansible_mosquitto_settings.password_file must be defined if you want to enable users
```

Comment:
sansible_mosquitto_settings.password_file should be  sansible_mosquitto_settings_password_file (same for line 140)

Reply:
Thank you, I missed the README...